### PR TITLE
update et-data tors

### PR DESCRIPTION
**Pull request by amilan17**

### CODE REVIEW

File: et-tt/et-data.adoc [hunk 1]
```diff
@@ -3,8 +3,8 @@
 === Terms of Reference
 
 [loweralpha]
-. Maintain and further develop table-driven codes in the Manual on Codes (WMO-No. 306) Vol. I.2, to meet the requirements of all Members and partner organizations, 
+. Maintain and further develop table-driven codes in the Manual on Codes (WMO-No. 306) Vol. I.2, to meet the requirements of all Members, WMO Earth system disciplines and domains and partner organizations,
```
Comment (golfvert) [inline et-tt/et-data.adoc:6]: It is a matter of English... So, I am clearly not the best one for that.
`all Members, WMO Earth system disciplines and domains and partner organizations,` sounds weird to me.
The first `and` joins disciplines and domains. The second `and` being the link with the last item in the list.
Is it better to have this: `all Members, all WMO Earth system disciplines and partner organizations,` or `all Members, WMO Earth system disciplines and domains as well as partner organizations,`

**Review by golfvert — COMMENTED** [review]
English comment/question.

File: et-tt/et-data.adoc (hunk 1 above)
Comment (amilan17) [inline et-tt/et-data.adoc:6]:
```suggestion
. Maintain and further develop table-driven codes in the Manual on Codes (WMO-No. 306) Vol. I.2, to meet the requirements of all Members, WMO Earth system disciplines and domains as well as partner organizations,  
```

Comment (amilan17) [reply et-tt/et-data.adoc]: I updated the text

**Review by efucile — APPROVED** [review]

**Pull request merged**